libs/schematic/api-description/build/openapi.yaml

```diff
@@ -753,6 +753,45 @@ paths:
                 $ref: '#/components/schemas/TangledTreeText'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /generateGoogleSheetManifests:
+    get:
+      tags:
+        - ManifestGeneration
+      summary: Generates a list of google sheet links
+      description: Generates a list of google sheet links
+      operationId: generateGoogleSheetManifests
+      parameters:
+        - $ref: '#/components/parameters/schemaUrl'
+        - $ref: '#/components/parameters/addAnnotations'
+        - $ref: '#/components/parameters/datasetIdArray'
+        - $ref: '#/components/parameters/assetViewIdQuery'
```

Comment:
When generating a manifest, assetViewId is not required. For people who are generating a new manifest (without pulling any existing manifests), they don't need an asset view.

Reply:
You want to be an optional parameter correct?

Reply:
Yes. It needs to be optional